bedrock/mozorg/templates/mozorg/plugincheck.html

```diff
@@ -89,10 +85,23 @@
 {% block content %}
 <main role="main">
   <header id="main-feature">
-      <h1 class="large">{{_('Check Your Plugins')}}</h1>
+      <a href="{{ url('firefox') }}">{{ high_res_img('plugincheck/firefox-logo.png', {'alt': 'Mozilla Firefox', 'width': '145', 'class': 'firefox-logo'}) }}</a>
+      <h2 class="large">{{_('Check Your Plugins')}}</h2>
       <p class="preamble">{{_('Keeping your plugins up to date helps Firefox run safely and smoothly.')}}</p>
   </header>
 
+  <section class="newsletter-container">
+    {% if LANG == 'en-US' %}
```

Comment:
Is there a reason why this is `en-US` only, and other locales see a large block of empty space next to the newsletter?